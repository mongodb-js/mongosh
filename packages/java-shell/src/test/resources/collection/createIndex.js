// before
db.coll.remove({});
db.coll.insert({category: "cat1", v: 1});
db.coll.insert({category: "cat2", v: 2});
db.coll.insert({category: "cat2", v: 3});
// command
db.coll.createIndex({category: 1}, {collation: {locale: "fr"}});
// command
db.coll.getIndexes();
// clear
db.coll.drop();
