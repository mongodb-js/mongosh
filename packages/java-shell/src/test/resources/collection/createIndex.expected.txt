[ "category_1" ]
[ { "v": 2, "key": { "_id": 1 }, "name": "_id_", "ns": "admin.coll" }, { "v": 2, "key": { "category": 1 }, "name": "category_1", "ns": "admin.coll", "collation": { "locale": "fr", "caseLevel": false, "caseFirst": "off", "strength": 3, "numericOrdering": false, "alternate": "non-ignorable", "maxVariable": "punct", "normalization": false, "backwards": false, "version": "57.1" } } ]